tests/main/hook-permissions/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Ensure that post-refresh hook has permissions for its connected slot.
+
+details: |
+    Ensure that post-refresh hook has permissions of the connected
+    network-manager slot, and can introspect network-manager dbus
+    interface.
+
+systems: [ubuntu-core-*]
+
+prepare: |
+    snap install network-manager
```

Comment:
@stolowski thanks for the test. Is it possible to use a different snap instead of network-manager?
Some time ago we had to disable some tests on the boards arm because there were problems when network-manager was installed and also netplan is installed in the system.

Reply:
I think so, the point is to check dbus introspection, should be doable with other dbus-based interfaces, I'll try something else, thanks.

Reply:
I've changed it to use upower-observe.